src/generator.cs

```diff
@@ -5854,7 +5988,7 @@ public void Generate (Type type)
 						print ("return Dlfcn.GetSizeF (Libraries.{2}.Handle, \"{1}\");", field_pi.Name, fieldAttr.SymbolName, library_name);
 					} else if (field_pi.PropertyType == typeof (long)){
 						print ("return Dlfcn.GetInt64 (Libraries.{2}.Handle, \"{1}\");", field_pi.Name, fieldAttr.SymbolName, library_name);
-#if !WATCH
+#if HAVE_COREMEDIA && HAVE_AVFOUNDATION
```

Comment:
`&&` because the code inside requires both AVFoundation and CoreMedia to work (I figured that's the closest to the original ifdef).

The logic itself might be wrong, but I wanted to change the logic as little as possible.